user-dao-intro.adoc

```diff
@@ -0,0 +1,144 @@
+
+The Bisq DAO beats this tradition with collective decision-making on strategy and other matters through **weighted voting** based on BSQ stake.
+
+Here's how it works: any stakeholder can make a proposal in the DAO. It can be anything: a change in a trading parameter, a new bonded role, or even something more generic like an adjustment of overall project strategy. Stakeholders vote on the proposal, and their voting weight is based on BSQ stake, through a combination of two metrics:
+
+1. amount of BSQ committed to a particular vote
+2. amount of BSQ earned over time through contributions
+
+Taking both metrics into account discourages deep-pocketed whales from suddenly seizing control of the project, while still valuing dedicated stakeholders with consistent contributions over time. It brings about a **strict meritocracy** in which people need to somehow _buy in_ to the Bisq project in order to take part in its governance, and the more significant their stake, the stronger their voice.
+
+In this way, there is no need to rely on a single leadership team for direction: the community collectively manages itself.
+
+=== Ensure honesty in high-trust roles
+
+Despite the Bisq project's attempts to resist concentrating control as much as possible, it's impossible to avoid in some places. Domain name owners, social account admins, mediators, various node operators: these are all roles that must exist, but necessarily retain significant control and require a high degree of trust.
+
+Part of the benefit of a centralized team of thoroughly-vetted people reliant on a paycheck, as is the case in most companies, is that the risk of trusting people with significant responsibility is lower: they have a lot to lose if the company finds they have violated their integrity and engaged in foul play.
+
+This dynamic can be reproduced—at least partly—in a project without a central authority through **bonding**. The concept is simple enough: create skin in the game. Require that a person interested in taking on a high-trust role post a bond that's high enough to discourage them from engaging in foul play.
+
+But what happens if that person goes rogue? In a project without central authority, who decides when they've crossed the line, and what their fate should be?
+
+As with strategy and compensation, the community decides through voting. Anyone who suspects foul play can make a case for confiscating a bond with a new proposal, and stakeholders vote to determine an outcome.
+
+NOTE: Confiscating a bond is a harsh penalty which should not be taken lightly. Therefore, the Bisq DAO makes confiscation proposals especially hard to approve. Whereas regular proposals cost 1 BSQ to make, a confiscation proposal costs 100 BSQ to make, and it requires a quorum of at least 100,000 BSQ and 75% acceptance to pass (instead of the typical >50%).
+
```

Comment:
"Whereas regular proposals cost 1 BSQ to make, a confiscation proposal costs 100 BSQ to make,"
That is wrong. Fee is always the same.
Quorum is atm 200 000 BSQ and threshold 85%.

Reply:
Oops, I saw it [here](https://docs.bisq.network/dao/specification.html#confiscation). Is that outdated now? Or maybe I understood it wrong.

Reply:
Oh, yes that is outdated. Long time not reviewed it.... on my long todo list ;-)